### PR TITLE
[update] Update auth manifest metadata limit to 128

### PR DESCRIPTION
**Pull request by mhatrevi**

### CODE REVIEW

File: runtime/src/packet.rs [hunk 1]
```diff
@@ -25,7 +25,7 @@ pub struct Packet {
     pub len: usize, // Length in bytes
 }
 
-const MAX_PAYLOAD_SIZE: usize = 2050; // in dwords
+const MAX_PAYLOAD_SIZE: usize = 3586; // in dwords
```
Comment (nquarton) [inline runtime/src/packet.rs:28]: We'd probably want to test this against any new stack overflow detection features being developed.

**Review by nquarton — DISMISSED** [review]

**Review by ArthurHeymans — DISMISSED** [review]

File: runtime/README.md [hunk 2]
```diff
@@ -738,10 +738,8 @@ Command Code: `0x4154_4D4E` ("ATMN")
 | metadata\_vendor\_LMS\_sig    | u32[1344]    | Metadata Vendor LMOTS-SHA192-W4 signature                                   |
 | metadata\_owner\_ecc384\_sig  | u32[24]      | Metadata Owner ECC384 signature                                             |
 | metadata\_owner\_LMS\_sig     | u32[1344]    | Metadata Owner LMOTS-SHA192-W4 signature                                    |
-| metadata\_header\_revision    | u32          | Revision of the metadata header                                             |
-| metadata\_header\_reserved    | u32[3]       | Reserved                                                                    |
 | metadata\_entry\_entry\_count | u32          | number of metadata entries                                                  |
-| metadata\_entries             | MetaData[16] | The max number of metadata is 16 but less can be used                       |
+| metadata\_entries             | MetaData[128] | The max number of metadata is 16 but less can be used                      |
```
Comment (mtimkovich) [inline runtime/README.md:742]: Description should say 128 too

File: runtime/tests/runtime_integration_tests/test_set_auth_manifest.rs [hunk 3]
```diff
@@ -134,7 +134,7 @@ fn test_set_auth_manifest_cmd() {
 }
 
 #[test]
-fn test_set_auth_manifest_cmd_invalid_len() {
+fn test_set_auth_manifest_cum_invalid_len() {
```
Comment (mtimkovich) [inline runtime/tests/runtime_integration_tests/test_set_auth_manifest.rs:137]: typo?

File: auth-manifest/gen/src/generator.rs [hunk 4]
```diff
@@ -47,9 +47,7 @@ impl<Crypto: ImageGeneratorCrypto> AuthManifestGenerator<Crypto> {
         let slice = config.image_metadata_list.as_slice();
         auth_manifest.image_metadata_col.image_metadata_list[..slice.len()].copy_from_slice(slice);
 
-        auth_manifest.image_metadata_col.header.entry_count =
-            config.image_metadata_list.len() as u32;
-        auth_manifest.image_metadata_col.header.revision = 0; // [TODO] Need to update this.
```
[the remark this answers is not in the document]
Comment (mhatrevi) [reply auth-manifest/gen/src/generator.rs:52]: I removed this as it does not make to have two versions in a single structure. The auth manifest has a single version and the metadata list is part of the auth manifest.

**Review by jhand2 — APPROVED** [review]

File: runtime/src/packet.rs (hunk 1 above)
Comment (mhatrevi) [reply runtime/src/packet.rs:28]: Agreed.

**Review by jhand2 — APPROVED** [review]

**Review by nquarton — APPROVED** [review]

**Review by vsonims — APPROVED** [review]

**Pull request merged**